code/datums/traits/negative.dm

```diff
@@ -150,7 +150,7 @@
 			if("Scientist")
 				heirloom_type = /obj/item/toy/plush/slimeplushie
 			if("Roboticist")
-				heirloom_type = pick(subtypesof(/obj/item/toy/prize)) //look at this nerd
+				heirloom_type = pick(subtypesof(/obj/item/toy/prize), /obj/item/toy/plush/pkplush) //look at this nerd
```

Comment:
This doesn't work like might have hoped it would.
Will either return the new plush or the typecache, if it gets the cache, this will runtime.

Reply:
Thanks for letting me know! It should be fixed now